app/cafe.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
```

Comment:
You're missing the function annotation for the return type. According to the checklist, you should add it. Here is a good example: `def visit_cafe(self, visitor: dict) -> str:`.